lighthouse-core/audits/cache-start-url.js

```diff
@@ -37,21 +37,18 @@ class CacheStartUrl extends Audit {
    * @return {!AuditResult}
    */
   static audit(artifacts) {
-    let cacheHasStartUrl = false;
-    const manifest = artifacts.Manifest.value;
-    const cacheContents = artifacts.CacheContents;
-
-    if (!(manifest && manifest.start_url && manifest.start_url.value)) {
+    if (!artifacts.Manifest || !artifacts.Manifest.value) {
```

Comment:
this is checking two error states.
1) no manifest, so the `artifacts.Manifest` is `null`. This could do the explicit check, but not sure if it's worth it as the gatherer will only return `null` or an object, so only null will be falsy.

2) invalid JSON, so `artifacts.Manifest.value` will be `undefined`. Invalid JSON is the only way that `artifacts.Manifest.value`, but we have to check for it.

   One solution I considered to get rid of the extra check was to throw if the JSON was invalid, but it felt a bit weird to declare a gatherer error when it really was an error in the user's data. It would also mean that every manifest audit would have a `debugString` of `Audit error: Manifest gatherer encountered an error: invalid JSON` (or whatever) . That's a lot of visual noise when it could just be on the `manifest-exists` audit as it is here. OTOH, invalid JSON in your manifest is a pretty big and obvious error and one the site owner should fix quickly, so maybe that's fine.